src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  margin: 0 auto;
```

Comment:
Invalid BEM syntax. You can not define Block positioning, so margin should be removed.

You can try to create mix: placing all positioning logic of Block (your margin) on Element wrapper.
You can do smth like that:

```
<main class="main">  
  <section class="catalog main__catalog"> <------- here define position rules for .catalog via main__catalog element
```